### ISSUE TITLE
Folder Naming standard to help with Antivirus exceptions

### ISSUE THREAD

**Issue by Xeliaa** [ButzYung/SystemAnimatorOnline#78]
Aloha, wonder if you can start incorporating Folder naming that don't change other then topology ( what's created via a zip extract )
it will help for Antivirus's like Bitdefender and Avast for not hitting the executable or the JS scripts when they get activated each time.
Having a stable naming folder makes it so we don't have to exclude path and files again from each update, Just a thought Mahalo

**Comment by ButzYung** [comment]
Yeah those false positives from a few virus scanners are annoying, though I have tried my best to prevent them from happening. The latest version of XR Animator didn't trigger any warning the last time I checked with VirusTotal. Is there any virus scanner that is still triggered right now?
